pyproject.toml

```diff
@@ -91,8 +91,11 @@ test = [
 ]
 
 dev = [
-    "editables",
-    "vendoring; python_version ~= '3.8'",
+    "black>=23.3.0",
+    "editables>=0.3",
+    "isort>=5.11.5",
+    "pre-commit>=2.21.0",
+    "vendoring>=1.2.0; python_version ~= \"3.8\"",
```

Comment:
Wasn't sure if the versioning here wasn't set on purpose. I just used pdm to add the files. Let me know if you want me to change these out.